### PR TITLE
Add a couple of paragraphs and questions to CoC and privacy policy

### PR DESCRIPTION
**Pull request by danfishgold**
Sorry for dropping the ball on this. It's been a little hectic for me lately what with moving, getting a cold, and being glued to the news...

In any case, I've added some more content to the code of conduct. I'm not really sure how to word the privacy policy but I gave it a shot.

I've left some questions in both files (denoted by lines that start with `> `)

Feel free to change everything I wrote 😄

### CODE REVIEW

**Comment by katjam** [comment]
Thank you! No need to apologise for delay... especially given current events.

I'll take a look.

File: content/conduct.md [hunk 1]
```diff
@@ -3,8 +3,24 @@ title: Code of Conduct
 type: page
 ---
 
-# Code of conduct
+# Code of Conduct
+
+We've found that code reading clubs work best when you involve people of different backgrounds and levels of experience. People read code differently and we can learn a lot from each other! It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
```
Comment (katjam) [inline content/conduct.md:8]:
```suggestion
We've found that code reading clubs work best when you involve people with different backgrounds and levels of experience. People read code differently and we can learn a lot from each other. It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
```

Comment (katjam) [reply content/conduct.md:8]: Maybe reads better as with and one !

The tone and content is ace.

File: content/conduct.md [hunk 2]
```diff
@@ -3,8 +3,24 @@ title: Code of Conduct
...
 
 ## How to be with each other
```
Comment (katjam) [inline content/conduct.md:10]: All good ideas in this section. I'll have a go drafting something.

File: content/conduct.md [hunk 3]
```diff
@@ -3,8 +3,24 @@ title: Code of Conduct
...
+
 ## Guidelines for running a club
+
+The code of conduct above is the one we use in our clubs. Feel free to use it as is or modify it to fit your circumstances.
+
+As a facilitator it's important that you not only follow your code of conduct, but also enforce it. Your code of conduct should be made available to attendees before the club and you should ask them to read it before attending. It's good practice to quickly go over the guidelines at the beginning of the session. Make sure to give attendees a way to inform you of any code of conduct violations they experience.
+
 ## Talk to us
+
+> is this section for people in our clubs who need us to deal with a CoC issue, or for people who want to run a club and have questions?
```
Comment (katjam) [inline content/conduct.md:26]: I think I intended it as a section for people who want us to deal with a CoC issue or have a related concern / suggestion.

File: content/privacy.md [hunk 4]
```diff
@@ -8,12 +8,19 @@ type: page
 ## Who we are
 
 ## Your personal information
+
 Your personal information belongs to you. Any personal information we get from you will only be used internally and never shared with third parties.
 
+### Contact information
+
+> If we ask for your email address we will only use it to contact you regarding sessions you signed up for, or to notify of future sessions.
```
Comment (katjam) [inline content/privacy.md:16]: Yes and we could add something about news.

Maybe `notify of future sessions` could become `notify you of future opportunities to engage with Code Reading Clubs`

File: content/privacy.md [hunk 5]
```diff
@@ -8,12 +8,19 @@ type: page
...
+
 ## Public code reading sessions
 
 ### Sharing tools
 
+In our remote public code reading sessions we use collaborative tools to allow us to talk to each other and to collect discussion points from attendees.
+
```
Comment (katjam) [inline content/privacy.md:23]: > I need help with this wording: 

In order to keep it simple, many of these resources are accessible on links which allow universal access. We will aim to make these private or remove them from the internet as soon as possible after a session, but sometimes, the artefacts from a session might remain online for a period of time.

**Review by katjam — COMMENTED** [review]
Some small steps forward.

I'll take a look at the "How to be with each other" section asap.

File: content/privacy.md (hunk 5 above)
Comment (danfishgold) [reply content/privacy.md:23]: I'm not sure what you mean by artefacts so maybe some examples would be good here.

Other than that, what do you think about these changes:

To keep it simple, we may use tools that are accessible using public links. After each session we will aim to make these private or remove them from the internet, but sometimes artefacts from a session might remain online for a period of time.

File: content/conduct.md (hunk 1 above)
Comment (danfishgold) [reply content/conduct.md:8]: I drafted this after watching a !!con talk so I was in a heightened exclamation-mark mood 😅

File: content/privacy.md (hunk 5 above)
Comment (katjam) [reply content/privacy.md:23]: Thanks - I tweaked and used that.

**Review by katjam — APPROVED** [review]
I'm happy for this to go in as it is. I know it is incomplete - but better to have something up than keep trying to get it all done in one go.

**Pull request merged**